### ISSUE TITLE
No visual keyboard focus is present around the radio buttons in Feedback page of preferences in High contrast black mode

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#356]
### Steps to Reproduce

Pre-Requisite: Change the system high contrast mode to black before launching the application.

1. Launch Xamarin Workbooks application.
2. Navigate to Tools and Preferences. Invoke preferences tab. It will open the general tab of preferences. Invoke the Feedback tab item using mouse (As tab items are not accessible by keyboard)
3. Navigate to "Yes, I am willing to participate (Recommended)/ No, I am not willing to participate) radio buttons.
### Expected Behavior
Visual keyboard focus should be visible upon navigating to radio buttons
### Actual Behavior
No visual keyboard focus is present around the radio buttons in Feddback page of preferences in High contrast black mode
Note : In high contrast white mode dotted rectangular focus appears around the radio buttons
### User Impact
User will not know whether the radio buttons got the focus or not.
### Screenshots
![12 hc black](https://user-images.githubusercontent.com/22956130/40723987-a511fa04-643d-11e8-8d99-67806f38293c.PNG)
![12 hc white](https://user-images.githubusercontent.com/22956130/40723988-a5460ace-643d-11e8-8b66-cf2f2ed3182e.PNG)
![12 normal mode](https://user-images.githubusercontent.com/22956130/40723989-a5721876-643d-11e8-9a54-514608c73d7b.PNG)

### Environment
Windows : Version 1803 (OS Build : 17134.81)
Xamarin Version : 1.4(master/b3f92f9)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS4.3.1
